src/styles/global.sass

```diff
@@ -0,0 +1,82 @@
+/*! Default styles from your beloved manGoweb (http://mangoweb.cz) */
+
+*, *:before, *:after
+	box-sizing: border-box
+
+html
+	font-size: $base-font-size
+
+body
+	background: $text-back
+	color: $text-color
+	font-family: $font-family-base
+	font-size: $base-font-size
+	line-height: $gratio
+	overflow-x: hidden
+	padding: 1em
+
+a
+	color: $c-main
+	text-decoration: underline
+
+h1, h2, h3, h4, h5, h6
+	font-family: $font-family-head
+	line-height: 1
+	margin: $spacing*2 0 1em
+	text-rendering: optimizeLegibility
+
+img
+	max-width: 100%
+	height: auto
+	vertical-align: middle
+
+p
+	margin: 0 0 $spacing
+
+	img
+		margin-bottom: $spacing
+
+ol, ul, li
+	margin: 0
+	padding: 0
+
+blockquote
+	margin: $spacing 0
+	padding: $padding 0 $padding $spacing
+	border-left: 2px solid
+	color: $text-color-light
+
+table
+	width: 100%
+
+input,
+button,
+select,
+textarea
+	font-family: inherit
+	font-size: inherit
+	line-height: inherit
+
+svg
+	pointer-events: none
+
+b, i
+	font-style: normal
+	font-weight: normal
+
+
+// Helper classes
+
+.accessible
+	position: absolute !important
+	clip: rect(1px 1px 1px 1px) /* IE6, IE7 */
+	clip: rect(1px, 1px, 1px, 1px)
+	overflow: hidden
+	size: 1px 1px
+.fleft
```

Comment:
a .accessible ti nevadí? Je to stejný princip utilitek, které se nevyplatí psát přímo do komponenty